alf/layers.py

```diff
@@ -462,6 +462,18 @@ def make_parallel(self, n: int):
         """
         return ParallelFC(n=n, **self._kwargs)
 
+    def __repr__(self):
```

Comment:
Perhaps try @alf.repr_wrapper?

Reply:
> Perhaps try @alf.repr_wrapper?

Updated to use repr_wrapper. The only downside is that sometimes the repr str can get really long.